.github/workflows/kickstart-tests.yml.j2

```diff
@@ -294,7 +310,7 @@ jobs:
         if: always()
         uses: actions/upload-artifact@v4
         with:
-          name: 'logs'
+          name: kstest_logs_${{ needs.pr-info.outputs.kstest_run_description }}
```

Comment:
This is a different job than PR info. You need kstest_logs_${{ needs.pr-info.outputs.timestamp }}

Or in line 81 you want to set the correct name:
Instead of :
timestamp: ${{steps.kstest_run_description.outputs.image_description}}
you want:
kstest_run_description: ${{steps.kstest_run_description.outputs.image_description}}